### ISSUE TITLE
Training details

### ISSUE THREAD

**Issue by rulixiang** [NVlabs/SegFormer#37]
Hi, I'm trying to reproduce SegFormer on PASCAL VOC dataset. When using the codes of this repo, I could get `~77%` mIoU (without `multi-scale test`). However, I only got `~75%` mIoU with my reproduced code. Here are my training details.

I have reproduced the training and validation data pipeline, including `random scaling` , `random horizontal flipping` , and `random cropping`, etc. For the model, I used the code of this repo and the pre-trained weights. I also used an `AdamW` optimizer with a `warmup` scheduler. The other optimizer settings are set as the same with this repo.

Therefore, I'm wondering if there are any extra training details in SegFormer or `mmseg` itself. I'll very appreciate for your reply.

**Comment by Hugstar** [comment]
I've encountered the same issue. 
What are the issues that you're seeing when training with your reproduced code? 
Is that related to the edges sharpness?

**Comment by xieenze** [comment]
Hi,
I believe all the training detail should be contained in the `dataset config`. But I am not sure whether your implementation is exactly the same as mmseg's.

**Comment by rulixiang** [comment]
Thanks for your reply.
